test/network.spec.js

```diff
@@ -459,6 +459,18 @@ module.exports.addTests = function({testRunner, expect}) {
       await page.goto(server.EMPTY_PAGE);
       expect(error.message).toContain('Request Interception is not enabled');
     });
+    it('should work with file URLs', async({page, server}) => {
+      await page.setRequestInterception(true);
+      const urls = new Set();
+      page.on('request', request => {
+        urls.add(request.url().split('/').pop());
+        request.continue();
+      });
+      await page.goto('file://' + path.join(__dirname, 'assets', 'one-style.html'));
```

Comment:
Path.join will have the wrong way slashes on windows

Reply:
I wonder how did it pass on appveyor.

Reply:
I think Chrome auto corrects slashes. A bit iffy to rely on that though.